# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/hittest/browser_test_general.js
# Commit: c552d41490c7
# Full Hash: c552d41490c7670eb12076a3028b6863c23f4ff0
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2023-06-13 03:42:25
# Regressor Bug: 1837030
# File Overlap Count: 1
# Description:
#   Bug 1837030: Treat fuzzily-matched text leaves as valid in OffsetAtPoint r=Jamie,nlapre
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180515
# ==============================================================================

diff -r c33c010f3a4f -r c552d41490c7 accessible/tests/browser/hittest/browser_test_general.js
--- a/accessible/tests/browser/hittest/browser_test_general.js	Mon Jun 12 20:31:07 2023 +0000
+++ b/accessible/tests/browser/hittest/browser_test_general.js	Mon Jun 12 21:01:04 2023 +0000
@@ -302,6 +302,14 @@
       generic, // Direct Child
       invisible.firstChild // Deepest Child
     );
+
+    await testOffsetAtPoint(
+      findAccessibleChildByID(docAcc, "invisible", [Ci.nsIAccessibleText]),
+      1,
+      1,
+      COORDTYPE_PARENT_RELATIVE,
+      0
+    );
   },
   { chrome: false, iframe: true, remoteIframe: true }
 );
